zds/forum/tests/tests.py

```diff
@@ -776,6 +776,51 @@ def test_frontend_alert_existence_other_pages(self):
         template_response = self.client.get(topic.get_absolute_url() + "?page=2")
         self.assertNotIn(expected, template_response.content.decode("utf-8"))
 
+    def test_frontend_topic_message_when_profile_readonly(self):
+        readonly_profile = ProfileFactory()
+        readonly_profile.is_active = True
+        readonly_profile.can_write = False
+        readonly_profile.save()
```

Comment:
À vérifier mais normalement : 
- tu peux préciser les valeurs que tu souhaites directement lors de l'initialisation ([la documentation de la bibliothèque `factory_boy`](https://factoryboy.readthedocs.io/en/stable/)) ;
- `is_active=True` ne sert à rien car il s'applique sur l'objet `User` et est déjà par défaut à `True` dans les `UserFactory` dont dépend `ProfileFactory` (https://github.com/SpaceFox/zds-site/blob/11ad4e038050613a46810c180d3b1e520457b3d5/zds/member/factories.py#L25)

```suggestion
        readonly_profile = ProfileFactory(can_write=False)
```

Reply:
Je confirme que ça fonctionne.